features/filiatie.feature

```diff
@@ -0,0 +1,618 @@
+# language: nl
+
+Functionaliteit: Als gemeente wil ik inzicht in de filiatie van Kadastraal onroerende zaken
+    Zodat ik inzicht heb in de splitsingen en samenvoegingen waarbij de kadastraal onroerende zaak betrokken is geweest.
+    En zodat ik inzicht heb in alle splitsingen en samenvoegingen die hebben plaatsgevonden tot en met de uitgifte van de grond door de gemeente.
+
+    Filiatie is de relatie naar de kadastraal onroerende zaken waaruit betreffende onroerende zaak is ontstaan en/of waarin de kadastraal onroerende zaak is overgegaan.
+    
+    Er zijn twee soorten kadastraal onroerende zaken, een perceel en een appartementsrecht. Een perceel is een begrensd deel van het Nederlands grondgebied. Een appartementsrecht omvat de bevoegdheid tot het gebruik van bepaalde gedeelten van een gebouw die blijkens hun inrichting bestemd zijn of worden om als afzonderlijk geheel te worden gebruikt. 
+    
+    De filiatie van een perceel wordt op een andere manier geregistreerd en daardoor ook op een andere manier in de API getoond dan een appartementsrecht.
+
+    Een perceel kan worden gesplitst in meerdere nieuwe percelen. Een perceel kan ook worden samengevoegd met één of meerdere andere percelen. 
+    Wanneer een perceel wordt gesplitst of samengevoegd vervallen de oude kadastraal onroerende zaken en ontstaan er nieuwe kadastraal onroerende zaken. De vervallen kadastraal onroerende zaken (percelen) zijn dan nog wel raadpleegbaar.
```

Comment:
Kan een perceel worden samengevoegd met appartementsrecht(en)? Deze vraag omdat in de tekst staat dat de oude kadastraal onroerende zaken vervallen en niet dat de oude percelen zijn vervallen

Reply:
ik zet erbij dat de onroerende zaken percelen zijn